pages/trackingprotection.py

```diff
@@ -4,6 +4,8 @@
 
 class TrackingProtectionPage(Page):
 
+    URL_TEMPLATE = 'https://mozilla.github.io/tracking-test/'
```

Comment:
suggestion: move constant declaration above class and below imports per:
https://www.python.org/dev/peps/pep-0008/#constants

It's not law, just a guideline - see:
https://stackoverflow.com/questions/5096025/should-class-specific-constants-still-be-declared-at-module-level